bin/check_golint

```diff
@@ -37,7 +37,11 @@ if [ -z "$ARGUMENTS" ]; then
     ARGUMENTS=`go list ./...`
 fi
 
-MESSAGES_ERROR=`golint --set_exit_status --min_confidence=1 ${ARGUMENTS} 2>/dev/null`
+MESSAGES_ERROR=()
+for arg in ${ARGUMENTS[@]}; do
+    MESSAGES_ERROR+=`golint --set_exit_status --min_confidence=1 ${arg} 2>/dev/null`
```

Comment:
Thanks for the review.  First I guess we should work out why you are able to pass `golint` a list and I am unable to do so.  Not that it should matter but what version of `go` are you running? I'm doing this with 1.11

I can't really think of anything off the top of my head why this fails for me.  Can we leave this open for a few days and see if I work it out please?

Reply:
Sure, we have [stale policy](https://github.com/mysteriumnetwork/node/blob/master/.github/stale.yml#L22-L32). If PR is inactive for 2 weeks it will be marked as a `stale` and will be closed after more 2 weeks if no actions appeared. 

btw, I'm using `go1.11.1`.